docs/configuration/git-providers/monitoring-conf/loki-values.yaml

```diff
@@ -0,0 +1,24 @@
+resources:
+  limits:
+    cpu: 1
+    memory: 1000Mi
+  requests:
+    cpu: 500m
+    memory: 500Mi
+config:
+  limits_config:
+    cardinality_limit: 200000
+    ingestion_rate_mb: 15
+    ingestion_burst_size_mb: 20
+serviceMonitor:
+  enabled: true
+persistence:
+  enabled: true
+  size: 100Gi # adjust persistence volume size if needed
+  storageClassName: <--- storage class ---> # use, for instance microk8s-hostpath, for microk8s
```

Comment:
Wouldn't "default" be good enought here?

Ideally people wouldn't have to edit this field.

Reply:
yes, that would work